components/common/githubButton.js

```diff
@@ -0,0 +1,37 @@
+import { Button, Image, Link } from '@chakra-ui/core'
+import { defaultProps } from '../../utils/defaultBtnProps'
+
+import PropTypes from 'prop-types'
+
+const githubLink = 'https://github.com/login/oauth/authorize?client_id=b8a7923343a52c37269e&state=poopie&scope=repo%20read:org'
+
+const fetchGHLink = ({ login }) => {
```

Comment:
nit: would rename this bc `fetch` is an overloaded word; in fact, does this even need to be a function ? wondering if it would be more readable to just have:

```
const GitHubLoginButton = ({ children, login, ...props }) => {
  const githubLink = login ? ... : ...
  return (
    <Link href={githubLink}>
      ...
    </Link>
  )
}
```
anyways its a nit :)